stable/airflow/templates/configmap-env.yaml

```diff
@@ -14,28 +14,36 @@ data:
   POSTGRES_HOST: "{{ template "airflow.postgresql.fullname" . }}"
   POSTGRES_PORT: "{{ .Values.postgresql.service.port }}"
   POSTGRES_DB: "{{ .Values.postgresql.postgresDatabase }}"
+  {{- if .Values.redis.enabled }}
   ## Redis DB configuration
   REDIS_HOST: "{{ template "airflow.redis.host" . }}"
   REDIS_PORT: "{{ .Values.redis.master.port }}"
+  {{- end }}
+  {{- if eq .Values.airflow.executor "Celery" }}
   ## Flower PORT
   FLOWER_PORT: "5555"
+  {{- end }}
   # Configure puckel's docker-airflow entrypoint
   EXECUTOR: "{{ .Values.airflow.executor }}"
   FERNET_KEY: "{{ .Values.airflow.fernetKey }}"
   DO_WAIT_INITDB: "false"
   ## Custom Airflow settings
   AIRFLOW__CORE__DONOT_PICKLE: "{{ .Values.dags.doNotPickle }}"
+  {{- if eq .Values.airflow.executor "Celery" }}
   AIRFLOW__CELERY__FLOWER_URL_PREFIX: "{{ .Values.ingress.flower.path }}"
   AIRFLOW__CELERY__WORKER_CONCURRENCY: "{{ .Values.workers.celery.instances }}"
+  {{- end }}
   AIRFLOW__CORE__DAGS_FOLDER: "{{ .Values.dags.path }}"
   AIRFLOW__CORE__BASE_LOG_FOLDER: "{{ .Values.logs.path }}"
   AIRFLOW__CORE__DAG_PROCESSOR_MANAGER_LOG_LOCATION: "{{ printf "%s/%s" .Values.logs.path "dag_processor_manager/dag_processor_manager.log" }}"
   AIRFLOW__SCHEDULER__CHILD_PROCESS_LOG_DIRECTORY: "{{ printf "%s/%s" .Values.logs.path "scheduler" }}"
   AIRFLOW__WEBSERVER__BASE_URL: "{{ .Values.ingress.web.host }}{{ .Values.ingress.web.path }}"
   # Disabling XCom pickling for forward compatibility
   AIRFLOW__CORE__ENABLE_XCOM_PICKLING: "false"
+  {{- if eq .Values.airflow.executor "Celery" }}
```

Comment:
can you move into a single block with this condition?

Reply:
done